python-mini-projects/projects/Download_images_from_website/scrap-img.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+import requests as rq
+import os
+from bs4 import BeautifulSoup
+import time
+
+# path= E:\web scraping\chromedriver_win32\chromedriver.exe
+path = input("Enter Path : ")
+
+url = input("Enter URL : ")
+
+output = "output"
+
+
+def get_url(path, url):
+    driver = webdriver.Chrome(executable_path=r"{}".format(path))
+    driver.get(url)
+    print("loading.....")
+    res = driver.execute_script("return document.documentElement.outerHTML")
+
+    return res
+
+
+def get_img_links(res):
+    soup = BeautifulSoup(res, "lxml")
+    imglinks = soup.find_all("img", src=True)
+    return imglinks
+
+
+def download_img(img_link, index):
+    try:
+        extensions = [".jpeg", ".jpg", ".png", ".gif"]
+        extension = ".jpg"
+        for exe in extensions:
+            if img_link.find(exe) > 0:
+                extension = exe
+                break
+
+        img_data = rq.get(img_link).content
+        with open(output + "\\" + str(index + 1) + extension, "wb+") as f:
+            f.write(img_data)
+        
+        f.close()
+    except Exception:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```